dali/pipeline/executor/executor.cc

```diff
@@ -393,27 +401,53 @@ void Executor::PruneUnusedGraphNodes() {
       "data produced by the pipeline.");
 }
 
-void Executor::SetupWorkspacesForGraph(WorkspaceBlob *wsb) {
+void Executor::SetupOutputInfo(const OpGraph &graph) {
+  DeviceGuard g(device_id_);
+  pipeline_outputs_ = graph.GetOutputs(output_names_);
+  for (auto tid : pipeline_outputs_) {
```

Comment:
`tid` was previously referring to thread id, please name it `tensor_id` here

Reply:
Done, in all the places I found.